project_issue_task_sync/README.rst

```diff
@@ -0,0 +1,35 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=====================
+Sync Issues and Tasks
+=====================
+
+This module syncs assigned user and state from task to issue and viceversa.
+If a task is created and does not have a issue a related issue will be
+generated. Projects may or may not have this syncing function, the activation
+of sync between issues and task may be activated on the project form by
+selecting the checkbox "sync issues and tasks". By default projects that use
+tasks and issues will be synced.
```

Comment:
well you case is well served by full syncing too.  

Create issue (task created)
triage issue: 
          a) close it (issue closes too)\
          b) create a task (not needed , task is already there, just do whatever on the task, that is allways synced with the issue.

i think we save also a passage in the workflow by no having to create a task.
True, there is some specificity to this module, it is used internally, but overall it spawned from the idea that  issue and tasks are quite redundant logically, it seems to me like these to models are the result of an accidental duplication of effort. the whol aim of this module is to tidy that out.  there fore the full syncing. there is allways the option to unsync a project and do whatever you want.

also added to views:   
      On a synced project  tasks are readonly on the issue form and issues are read only on the task form.